modules/installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+// * installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-nutanix-failure-domains_{context}"]
+= Configuring failure domains
+
+Failure domains improve the fault tolerance of an {product-title} cluster by distributing control plane and compute machines across multiple Nutanix Prism Elements (clusters).
+
+.Prerequisites
+
+* You have an installation configuration file (`install-config.yaml`).
+
+.Procedure
+
+. Edit the `install-config.yaml` file and add the following stanza to configure the first failure domain:
++
+[source,yaml]
+----
+# ...
+platform:
+  nutanix:
+    failureDomains:
+    - name: <failure_domain_name>
+      prismElement:
+        name: <prism_element_name>
+        uuid: <uuid>
+      endpoint:
+        address: <address>
+        port: <port>
+      subnetUUIDs:
+      - <network_uuid>
+# ...
+----
++
+where:
+
+`<failure_domain_name>`:: Specifies a name for the failure domain.
+`<prism_element_name>`:: Optional. Specifies the name of the Prism Element.
+`<uuid>`:: Specifies the universally unique identifier (UUID) of the Prism Element.
+`<address>`:: Optional. Specifies the domain name or IP address of the Prism Element.
+`<port>`:: Optional. Specifies the port that is used to log into Prism Element.
+`<network_uuid`:: Specifies the UUID of the Prism Element network that contains the virtual IP addresses and DNS records that you configured. Only one subnet per {product-title} cluster is supported.
```

Comment:
Only one subnet per Failure Domain is supported.

Reply:
Done.